app/actions/service_route_binding_create.rb

```diff
@@ -43,19 +46,25 @@ def event_repository
         )
       end
 
-      def validate!(service_instance, route)
+      def validate_service_instance!(service_instance, route)
         not_supported! unless service_instance.route_service?
         not_bindable! unless service_instance.bindable?
         route_is_internal! if route.try(:internal?)
         space_mismatch! unless route.space == service_instance.space
-        already_exists! if route.service_instance == service_instance
-        already_bound! if route.service_instance
+        already_bound! if route.service_instance && route.service_instance != service_instance
         return unless service_instance.managed_instance?
 
         service_instance_not_found! if service_instance.create_failed?
         operation_in_progress! if service_instance.operation_in_progress?
       end
 
+      def validate_binding!(binding, _route, _service_instance)
```

Comment:
You should remove the unneeded parameters.

Reply:
Solved in #3469